tools/applicationinsights-web-snippet/src/snippet.ts

```diff
@@ -235,7 +247,18 @@ declare var cfg:ISnippetConfig;
                 return scriptElement;
             }
 
-            let theScript = _createScript();
+            let theScript = _createScript(targetSrc);
+
+            if (appInsights.cr && loadFailed && domains.some(endpoint => targetSrc.indexOf(endpoint) !== -1)){
```

Comment:
We should not use `some` as it's not supported on IE8, so this would cause the snippet to throw an error and fail.